mlem/cli/main.py

```diff
@@ -259,6 +257,22 @@ def get_command(self, ctx: Context, cmd_name: str) -> Optional[Command]:
                 return cmd
         return None
 
+    @property
+    def help(self):
+        ctx = click.get_current_context()
```

Comment:
Looks familiar. Can you de-duplicate by moving to `MlemMixin`?

Reply:
Did what I could 🤔